package.json

```diff
@@ -83,6 +84,7 @@
     "es6-promisify": "~5.0.0",
     "express": "~4.16.4",
     "express-rate-limit": "~5.5.1",
+    "fast-xml-parser": "~4.0.1",
```

Comment:
I believe fast-xml-parser is not being used in eGauge; although it was at first. So it can removed from package.json and package-lock.json.

Reply:
I could not find any usage either so I removed.